.github/CODEOWNERS

```
@@ -1,40 +1,40 @@
 * @aws/aws-sdk-js-team
 
 # These packages are used by Smithy generated code
-packages/config-resolver/* @aws/smithy
-packages/eventstream-serde-browser/* @aws/smithy
-packages/eventstream-serde-config-resolver/* @aws/smithy
-packages/eventstream-serde-node/* @aws/smithy
-packages/fetch-http-handler/* @aws/smithy
-packages/hash-blob-browser/* @aws/smithy
-packages/hash-node/* @aws/smithy
-packages/hash-stream-node/* @aws/smithy
-packages/invalid-dependency/* @aws/smithy
-packages/md5-js/* @aws/smithy
-packages/middleware-apply-body-checksum/* @aws/smithy
-packages/middleware-content-length/* @aws/smithy
-packages/middleware-endpoint/* @aws/smithy
-packages/middleware-retry/* @aws/smithy
-packages/middleware-serde/* @aws/smithy
-packages/middleware-stack/* @aws/smithy
-packages/node-http-handler/* @aws/smithy
-packages/protocol-http/* @aws/smithy
-packages/querystring-builder/* @aws/smithy
-packages/service-client-documentation-generator/* @aws/smithy
-packages/smithy-client/* @aws/smithy
-packages/types/* @aws/smithy
-packages/url-parser/* @aws/smithy
-packages/util-base64/* @aws/smithy
-packages/util-base64-browser/* @aws/smithy
-packages/util-base64-node/* @aws/smithy
-packages/util-body-length-browser/* @aws/smithy
-packages/util-body-length-node/* @aws/smithy
-packages/util-defaults-mode-browser/* @aws/smithy
-packages/util-defaults-mode-node/* @aws/smithy
-packages/util-endpoints/* @aws/smithy
-packages/util-middleware/* @aws/smithy
-packages/util-retry/* @aws/smithy
-packages/util-stream-browser/* @aws/smithy
-packages/util-stream-node/* @aws/smithy
-packages/util-utf8/* @aws/smithy
-packages/util-waiter/* @aws/smithy
+/packages/config-resolver/ @aws/smithy @aws/aws-sdk-js-team @aws/smithy
```

`@aws-smithy` is added twice. Is that on purpose?
```suggestion
/packages/config-resolver/ @aws/aws-sdk-js-team @aws/smithy
```

---

Thanks for catching that, it is a mistake. Pushed a fix.